src/settings/index.ts

```diff
@@ -38,7 +38,8 @@ export const PRIVACY_POLICY = SPARKLE_PRIVACY_POLICY;
 
 export const SPARKLE_ICON = "/sparkle-icon.png";
 export const DEFAULT_MAP_BACKGROUND = "/maps/Sparkle_Field_Background.jpg";
-export const DEFAULT_VENUE_BANNER = "/assets/Default_Venue_Banner.png";
+export const DEFAULT_LANDING_BANNER = "/assets/Default_Venue_Banner.png";
+export const DEFAULT_VENUE_BANNER = "#000000";
```

Comment:
`DEFAULT_VENUE_BANNER` -> `DEFAULT_VENUE_BANNER_COLOR`

by the way, do we really need to store that color, can't we leave the background empty? instead of overwriting it to black?

Reply:
renamed
i've removed `DEFAULT_VENUE_BANNER` from venue creation, so that we're leaving it empty now 208fc95